.github/workflows/ci_integration.yml

```diff
@@ -260,12 +261,6 @@ jobs:
         uses: "docker/setup-qemu-action@v1"
       - name: "Set up Docker Buildx"
         uses: "docker/setup-buildx-action@v1"
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v1
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
These changes are to prevent pushing these containers to GH, and the above to trial run the build performance. Only the Dockerfile change is intended.